.github/workflows/git-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+name: gitleaks
+on: [pull_request, push, workflow_dispatch]
+jobs:
+  gitleaks-scan:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run gitlakes docker	    
+      uses: docker://zricethezav/gitleaks
+      with:
+        args: detect --source /github/workspace/ --no-git --verbose
```

Comment:
Github is showing a "no newline at tend of file" warning here.  Can your editor be configured to automatically "insert final newline" at the end of new files?  Here's a discussion about how to do it in VS Code:

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
@PhilR8 I added the final new lines as suggested, but now the github check failed for e2e testing, it says that cypress failed

Reply:
The failure is unrelated to your work here -- we're taking care of it in another story.  Thank you for making these changes!